core/lib/generators/spree/mailers_preview/templates/mailers/previews/shipment_preview.rb

```diff
@@ -0,0 +1,5 @@
+class ShipmentPreview < ActionMailer::Preview
+  def shipped_email
+    Spree::ShipmentMailer.shipped_email(Spree::Shipment.shipped.first || Spree::Shipment.first)
```

Comment:
```suggestion
    Spree::ShipmentMailer.shipped_email(Spree::Shipment.shipped.first)
```

this can lead to some 50x errors, so we should always preview shipped shipments